Utilities/bootstrap

```diff
@@ -799,26 +793,6 @@ def get_swiftpm_flags(args):
             "-Xlinker", "@executable_path/../lib/swift/pm/llbuild",
         ])
 
-    # Add a relative rpath to find Swift libraries in toolchains.
-    # On non-Darwin platforms, a relative rpath is necessary because Swift
-    # libraries are not part of the OS.
```

Comment:
I think this comment should actually stay since we're only changing this for Darwin.

Reply:
Ah I had automerge enabled, going to restore this comment in a follow up.

Reply:
Actually, we moved the non-Darwin part out of bootstrap and put it behind `SWIFTCI_INSTALL_RPATH_OS`, so this it is correct for the comment to be gone.